src/app/core/store/content/viewcontexts/viewcontexts.selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { createSelector } from '@ngrx/store';
+
+import { CallParameters } from 'ish-core/models/call-parameters/call-parameters.model';
+import { createContentPageletEntryPointView } from 'ish-core/models/content-view/content-view.model';
+import { getContentState } from 'ish-core/store/content/content-store';
+
+import { serializeContextSpecificViewContextId, viewcontextsAdapter } from './viewcontexts.reducer';
+
+const getViewcontextsState = createSelector(getContentState, state => state.viewcontexts);
+
+export const getViewcontextsLoading = createSelector(getViewcontextsState, state => state.loading);
+
+export const { selectEntities: getViewcontextEntities } = viewcontextsAdapter.getSelectors(getViewcontextsState);
+
+const getViewcontextInternal = (viewContextId: string, callParameters: CallParameters) =>
```

Comment:
Maybe add comment, that this selector is output-memoized. Alternatively rename it to `getViewcontextMemoized` (prefer the latter) 😉